### ISSUE TITLE
Separate questions for is-above-duration-threshold

### ISSUE THREAD

**Issue by May-N** [Siteimprove/alfa#177]
I didn't realise there would be a follow up question in R48.
It goes:
1. has-audio
2. is-above-duration-threshold
3. is-below-audio-duration-threshold

In order to minimise confusion, I think we need separate questions for video and audio.
"Does the element have a duration of more than 3 seconds?" is just too open to interpretation (I'd be thinking, does "element" refer to the audio or the audio track - and then second guess myself with the follow up question).

We could have:
- Is the video more than 3 seconds long?
- Is the audio track more than 3 seconds long? 
+ help text

... Then ask a single question for is-below-audio-duration-threshold: 
Can the audio be heard for more than 3 seconds?

**Comment by kasperisager** [comment]
@May-N Come to think of it, it may be better asking how long the video/audio is? What are your thoughts, @Jym77?

**Comment by May-N** [comment]
@kasperisager so like, asking the user to input the number of minutes/seconds? Could be cool

**Comment by Jym77** [comment]
Yeah, I do like the idea of asking the duration directly rather than asking if it's more or less than a (somewhat) arbitrary threshold. It's kinda more modular (what if another rule needs to know if audio is more than 2 seconds?)

This would, however, require bigger changes in Alfa and Dory (and frontend) as it is a new type of question (answer is not boolean/node/nodes). So we might want to discuss that also with the rest of the team and prioritise accordingly.
